### ISSUE TITLE
Visualization of beam tet and escher mesh and ODE convergence  

### ISSUE THREAD

**Issue by Asadanees** [mfem/mfem#289]
@v-dobrev and @tzanio @mlstowell 
Hope you are fine.
I have a problem from last two month I could not fix it.
First the colour of beam tet and escher mesh is only in yellow colour  after visualisation 
I think that there is some problem in ODE solver Mult. (same problem in parallel   @code)
I am implemented Linear Maxwell equations.
Could I send the send the code to you via email.
THanks and best wishes
regards,

Asad anees

**Comment by tzanio** [comment]
Hi Asad,

Can you attach the code in this issue (or maybe just paste the relevant sections)? It sound like you may not be sending the solution grid function (in addition to the mesh), see https://github.com/mfem/mfem/blob/master/examples/ex1.cpp#L189

Tzanio

**Comment by Asadanees** [comment]
I am just writing comments for you to that it is easy for you to understand.
I shall send you all the code for simple example.
Later I will send you other code linear and  non-linear examples.  
I have written three example One is parallel 
All this code will also property of MFEM. Moreover I also welcome you to joint publication.
I will write myself . you just prove and guide me to correct the code. I need only this contribution from you.
we will publish in collaborations  (jointly).
Thanks 
@tzanio and other mfem teams

**Comment by Asadanees** [comment]
[maxwell.tar.gz](https://github.com/mfem/mfem/files/1335096/maxwell.tar.gz)
feel free to contact if you have any questions?
thanks and best wished

**Comment by Asadanees** [comment]
Dear @tzanio ,
I have written in the start of code about the problem with title problem 1, problem 2, problem 3.
it is easy to understand the problems
Sincerely yours 
asad anees

**Comment by tzanio** [comment]
Hi Asad,

The color is only yellow, because the field is zero, you can see that on the colorbar (key `c` in glvis).

I took a quick look at the code, and there seem to be some issues -- for example `w` is used in `Maxwell::Mult` both as ND and RT vector, which can't be right :). Can you go carefully through the `Maxwell` class and verify that all vectors and matrices are as you want them?

Hope this helps,
Tzanio

**Comment by Asadanees** [comment]
Dear @tzanio 
Hope you are fine.
Thanks for your help. You are absolutely right. I shall send you soon right solution.
Than you can add examples &miniaps.
Best wishes

Asad anees

**Comment by Asadanees** [comment]
@tzanio @v-dobrev @mlstowell 
Hope you are fine.
Could you please provide the exact solution of Maxwell equations, if LLNL (you) have?
I want to implementin my next pull request? 
Thanks 
Anees

**Comment by tzanio** [comment]
You can just use a manufactured solution as in Example 3, see

https://github.com/mfem/mfem/blob/125ff226e4a8c50fb3aa9a77e871d41bc6b4b60b/examples/ex3.cpp#L211-L241

**Comment by Asadanees** [comment]
Dear @tzanio  and @v-dobrev ,
I want to sent you pdf file of numerical result and behavior of numerical results.
This is my paper but numerical result is very week. 
Numerical result are obtained when exact solution of Maxwells equations are given.
We can also consider a bench Mark for our future work.
Could I send an email  of my paper on you LLNL -email, where you can find the behavior numerical result. Moreover I upload a the result a pull result.
Please tell me soon
Thanks 
asad

**Comment by Asadanees** [comment]
Dear @tzanio 
Moreover I will make a new pull request of time domain finite element method when exact solution is known. 
Thanks 
ASad

**Comment by Asadanees** [comment]
Dear @tzanio 
I want to send a pull request put I could not do.
Might be I have no excess.

**Comment by tzanio** [comment]
You were able to create one previously: https://github.com/mfem/mfem/pull/334... there shouldn't be any issues.

**Comment by Asadanees** [comment]
Dear @tzanio 
No there is no issue.  Additionally, I found a lot of questions in mimapp-> joule.cpp.
Now I have to send an email to you about the summary of issues and paper.
You can find the behavior of numerical simulation, when exact solution is known.
This could be bench mark problem for us.
Could you please speed review the soon? Even you can correct where you feel necessary.
Thanks 
asad
